SPEC.md

```diff
@@ -176,16 +210,16 @@ line of the file.
 
 Offsets, however are 0-based. A Position of `{ "offset": 4 }` represents the _fifth_ character in the file. Importantly, the `offset` is from the beginning of the file, not the beginning of a line. Newline characters (and all characters) count when computing an offset.
 
-### Contents
+#### Content
```

Comment:
No, that's just a mistake on my end, spurred in part by my confusion about the Content/Body thing ... I'll change it back.